dandischema/models.py

```diff
@@ -135,6 +135,13 @@ def __new__(cls, name, bases, dct):
 class DandiBaseModel(BaseModel, metaclass=DandiBaseModelMetaclass):
     id: Optional[str] = Field(description="Uniform resource identifier", readOnly=True)
 
+    """
+    def __init__(self, **kwargs):
+        if "schemaKey" not in kwargs:
+            kwargs["schemaKey"] = "MissingSchemaKey"
+        super().__init__(**kwargs)
+    """
+
```

Comment:
what are those docstrings with code left for? Delete:
```suggestion
```
or add some comment on their purpose?